lib/circuit.js

```diff
@@ -220,6 +228,9 @@ class CircuitBreaker extends EventEmitter {
     this.removeAllListeners();
     this.status.shutdown();
     this.hystrixStats.shutdown();
+    if (PrometheusMetrics && this.metrics) {
```

Comment:
I think this can be simpler.
```suggestion
    if (this.metrics) {
```
If `PrometheusMetrics` is `undefined`, this always will be as well.

Reply:
That's right. But I'm not sure/tested what happen with the webpack config